packages/@uppy/informer/src/TransitionGroup.js

```diff
@@ -264,17 +264,10 @@ class TransitionGroup extends Component {
   render ({ childFactory, transitionLeave, transitionName, transitionAppear, transitionEnter, transitionLeaveTimeout, transitionEnterTimeout, transitionAppearTimeout, component, ...props }, { children }) {
     // TODO: we could get rid of the need for the wrapper node
     // by cloning a single child
-    const childrenToRender = []
-    for (const key in children) {
-      if (children.hasOwnProperty(key)) {
-        const child = children[key]
-        if (child) {
-          const ref = linkRef(this, key),
-            el = cloneElement(childFactory(child), { ref, key })
-          childrenToRender.push(el)
-        }
-      }
-    }
+    const childrenToRender = Object.values(children).filter(Boolean).map(child => {
```

Comment:
![Screenshot from 2022-08-02 23-43-17](https://user-images.githubusercontent.com/1199054/182539594-a7487815-b7ca-4bb8-8be4-06fd9a135507.png)

@aduh95

Reply:
Lost the `key` here somewhere.

Reply:
I think I have a duplicate at my place, let me take a look.

Reply:
Fixed in 5fca61733d50c0f7392149b5924ef059296f200f.